main/test/ca/uwaterloo/flix/library/TestString.flix

```diff
@@ -2601,100 +2575,118 @@ def indexOfRight08(): Bool = String.indexOfRight("ab", "ab") == Some(0)
 def indexOfRight09(): Bool = String.indexOfRight("bc", "abc") == Some(1)
 
 /////////////////////////////////////////////////////////////////////////////
-// isPrefixOf                                                              //
+// startsWith                                                              //
```

Comment:
Can you check if there are already test cases for startsWith and endsWith? There might be

Reply:
Or did we just happen to have startsWith and endsWith implemented, but without any tests?

Reply:
If that is the case, we should of course keep the tests :)

Reply:
We had 3 test cases each for `startsWith` and `endsWith`. I "merged" those with the test cases for `isPrefixOf`/`isSuffixOf` (which have been removed).
I can see the git diff might look a bit weird due to my copy paste and argument order switch, but the process was as follows:
- We had the functions: `startsWith`, `endsWith`, `isPrefixOf`, `isSuffixOf`, `isInfixOf`
- I removed the `isPrefixOf` and `isSuffixOf` functions
- I renamed `isInfixOf` to `contains`
- I merged the test cases for `isPrefixOf`/`startsWith` and `isSuffixOf`/`endsWith`
- I renamed the test cases for `isInfixOf` to `contains`

Reply:
Excellent